src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,7 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+import './UserInfo.scss';
+
+export const UserInfo = ({ user: { email, name } }) => (
```

Comment:
You also might want to include some error handling to display a useful message in case the 'user' prop is null or undefined.